open-metadata-implementation/access-services/data-platform/data-platform-server/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/listeners/DataPlatformInTopicListener.java

```diff
@@ -82,23 +91,37 @@ public void processEvent(String eventAsString) {
                             informationViewAsset.getRelationalDbSchemaType().getGUID(),
                             view.getViewEntity().getGUID(),
                             new InstanceProperties());
-                    event.getTableSource().setGuid(view.getViewEntity().getGUID());
+                    newViewEvent.getTableSource().setGuid(view.getViewEntity().getGUID());
                 }
-                eventPublisher.sendEvent(event);
-            } catch (Exception e) {
-                log.error("Exception processing event from in topic", e);
-                DataPlatformErrorCode errorCode = DataPlatformErrorCode.PROCESS_EVENT_EXCEPTION;
-
-                auditLog.logException("processEvent",
-                        errorCode.getErrorMessageId(),
-                        OMRSAuditLogRecordSeverity.EXCEPTION,
-                        errorCode.getFormattedErrorMessage(eventAsString, e.getMessage()),
-                        e.getMessage(),
-                        errorCode.getSystemAction(),
-                        errorCode.getUserAction(),
-                        e);
+                eventPublisher.sendEvent(newViewEvent);
+                log.debug("invalid event schema");
+
+
             }
+        } catch (IOException e) {
+            DataPlatformErrorCode errorCode = DataPlatformErrorCode.PARSE_EVENT_EXCEPTION;
```

Comment:
For future PR the exception handling needs improvement as right now the exception it is only audit-logged.

Reply:
Agree. I will add it as TODO and including it in the next PR.